### PR TITLE
Clearer navigation buttons

### PR DESCRIPTION
**Pull request by mikeallbutt**
This closes issues #764

Screenshot 2019-10-27 00 37 38
Screenshot 2019-10-27 00 37 56
Screenshot 2019-10-27 00 38 24
Screenshot 2019-10-27 00 39 12

### CODE REVIEW

**Comment by mikeallbutt** [comment]
I revised the following in my code and it all seems fine.


```
FILE: ...build/DiscipleTools/disciple-tools-theme/dt-assets/parts/group-details.php
--------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
--------------------------------------------------------------------------------
 170 | ERROR | [x] Line indented incorrectly; expected at least 1 tabs, found 0
 181 | ERROR | [x] Line indented incorrectly; expected at least 1 tabs, found 0
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------


```
Screenshot 2019-10-28 10 10 23

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@ phpcs.phar
 
 */development-only/*
 
+package-lock.json
```
Comment (corsacca) [inline .gitignore]: @mikeallbutt package.json and package-lock.json can't be ignored.

Comment (mikeallbutt) [reply .gitignore]: Yes. I have fixed it now. I didn't realise that it was going to affect others. (I thought it was just my local development.)

Comment (corsacca) [reply .gitignore]: You'll need to commit the fix.

Comment (mikeallbutt) [reply .gitignore]: > You'll need to commit the fix.

Fixed and pushed.

**Pull request merged**